lib/xandra.ex

```diff
@@ -123,6 +137,10 @@ defmodule Xandra do
     * `:port` - (integer) the port of the Cassandra server to connect
       to. Defaults to `9042`.
 
+    * `:compressor` - (module) the compressor module to use for compressing and
+      uncompressing data. See the "Compression" section in the module
+      documentation. By default this option is not present.
```

Comment:
I think we do. We usually say "Defaults to ...", and I like to say "Not present by default" as well; it may be slightly redundant, but clarity is not hurting IMO.